### DISCUSSION TITLE
Last call for Hacktoberfest prize draw nominations

### DISCUSSION THREAD

**Discussion by C0staTin** [mindsdb/mindsdb#1714] [General]
Hi MindsDB Community,

This is the last week for [Hacktoberfest](https://mindsdb.com/hacktoberfest/?utm_medium=referral&utm_source=github&utm_campaign=github%20discussions), so please submit your PR's and [register](https://mindsdb.com/hacktoberfest/?utm_medium=referral&utm_source=github&utm_campaign=github%20discussions) for the SWAG and Prize ... we need to know where to send stuff! :upside_down_face:

We have some exciting news to share with you next Monday - anyone care to guess what it could be? 🤔

Thanks,
MindsDB team

**Comment by MichaelLantz** [comment]
Just want to deliver some personal thank you's. Although I've had enthusiasm and individual participation with open source projects in the past, this was my first time partaking in hacktoberfest and truly feeling welcomed into an open source community. Just want to give my personal thanks to the following folks for making this experience/opportunity awesome. 

@C0staTin, @ZoranPandovski, @paxcema, @George3d6

PS: Who would've thought I could find something productive and awesome on social media? Thanks to whoever made that marketing decision. ;)

**Reply by C0staTin** [reply]
thanks @MichaelLantz !
this is the best appreciation of our work :-)

**Comment by ZoranPandovski** [comment]
Thank you @MichaelLantz, we are happy to have you in MindsDB Community :bear: :raised_hands: .
